src/descriptor/tr.rs

```diff
@@ -244,7 +267,7 @@ impl<Pk: MiniscriptKey, Ext: Extension> Tr<Pk, Ext> {
         } else {
             let mut builder = TaprootBuilder::new();
             for (depth, ms) in self.iter_scripts() {
-                let script = ms.encode();
+                let script = ms.as_miniscript().unwrap().encode();
```

Comment:
In 899269acefeb534f140cfd4d7f86b66ad567b55f:

Can you add a FIXME comment here so we remember to fix it?

Reply:
Ah, nope, fixed in a later commit.